src/app.js

```diff
@@ -0,0 +1,83 @@
+import {promisify} from 'util';
+import {MongoClient} from 'mongodb';
+import {createLogger} from '@natlibfi/melinda-backend-commons';
+
+const setTimeoutPromise = promisify(setTimeout);
+
+export default async function ({mongoUri, mongoDatabaseAndCollections, pollTime}, momentDate) {
+  const logger = createLogger();
+  logger.info('Starting mongo cleaning');
+  const client = await MongoClient.connect(mongoUri, {useNewUrlParser: true, useUnifiedTopology: true});
+
+  await createSearchProcess(mongoDatabaseAndCollections);
+
+  await client.close();
+  if (momentDate === '2021-05-08') { // test escape
+    return;
+  }
+
+  logger.info(`Done${pollTime ? `, await ${pollTime / 1000 / 60 / 60}h till next restart` : ''}`);
```

Comment:
- this was quite hard to read
- maybe something like this could be easier to read? 
```
  logger.info(pollTime 
     ? `Done, await ${pollTime / 1000 / 60 / 60}h till next restart.` 
     : 'Done.'
   );
```